deployment/README_deploy.md

```diff
@@ -17,6 +17,7 @@ In the terminal, use `heroku` commands to build docker images locally and then p
 First, use the project login credentials to access heroku admin:
```

Comment:
Could you update this line to:

"First, use the project login credentials (username: recordexpungpdx) to access the heroku and docker admin accounts, respectively:"